<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestion d'Evalutation Vectorielle</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;You can add vector with the add vector button same for the add function.&lt;/p&gt;&lt;p&gt;To get the trajectory, the statistics and the list of vector you have to select one vector in the list of the same name and one function in the list of the same name and then click on execute.&lt;/p&gt;&lt;p&gt;To load a file just click on load file.&lt;/p&gt;&lt;p&gt;To save something do the same as for the execute button but at the end click on save file.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QListWidget" name="listWidget_vect">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>221</width>
      <height>201</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>V0 = (5.00000)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>L = (5.0000,2.00000)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>U = (14.00000,3.33450,-5.0000)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>W = (7.00000,2.00000,22.00000,1.00000)</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Button_excute">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;To dispay the list of vectors, the trajectory and the statictics.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Please select one vector and one funtion.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_funct">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>210</y>
      <width>221</width>
      <height>201</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>F(x) = (x+8)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>G(x,y) = (x+10;-1*y)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>T(x,y,z) = (2*x;-10+y;z*-1)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>F(x,y,z,a) = (x+1;y*2;z-x;a*-1)</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Button_erase_vect">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>420</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ERASE</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="SpinBox_nbr_vect">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How many vector do you want to be calculat.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::PlusMinus</enum>
    </property>
    <property name="prefix">
     <string>Number of vectors : </string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_addfunct">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add function&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Add Function</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_addvect">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add vector&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Add Vector</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_load">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Load file &lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;The initial vector will be add in the left list&lt;/p&gt;&lt;p&gt;The function will be add in the right list&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Load File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_erase_funct">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>420</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ERASE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_save">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;To save the list of vectors, the trajectory and the statictics.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Please select one vecteur and on function.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Save File</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>470</y>
      <width>591</width>
      <height>20</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;To get help click on the '?' button and click where you want to get help. This work on every window.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
